creator-node/src/snapbackSM/peerSetManager.js

```diff
@@ -213,23 +216,24 @@ class PeerSetManager {
       throw new Error('No discovery provider currently selected, exiting')
     }
 
-    const requestParams = {
-      method: 'get',
-      baseURL: this.discoveryProviderEndpoint,
-      url: `users/creator_node`,
-      params: {
-        creator_node_endpoint: this.creatorNodeEndpoint
-      },
-      timeout: DEFAULT_AXIOS_TIMEOUT_MS
-    }
-
     // Will throw error on non-200 response
     let nodePrimaryUsers
     try {
-      const resp = await axios(requestParams)
+      const resp = await Utils.asyncRetry({
```

Comment:
do we still need this legacy fn? cc @SidSethi

Reply:
nope! just added johannes to the card to remove it (https://linear.app/audius/issue/CON-74/delete-deprecated-getnodeprimaryusers-function-and-associated-route-in)